types/es-abstract/scripts/collect-intrinsics.ts

```diff
@@ -3,540 +3,68 @@
 import path = require('path');
 import fs = require('fs');
 
+import { BaseIntrinsic, Override, BASE_INTRINSICS, LEGACY_ALIASES, BASE_INTRINSIC_DATA } from './intrinsics-data';
+
 const OUT_FILE_PATH = path.resolve(__dirname, '..', 'GetIntrinsic.d.ts');
 
-const hasSymbols = true;
-const { getPrototypeOf: getProto, getOwnPropertyDescriptor: $gOPD } = Reflect as {
-    getPrototypeOf(target: object): object | null;
-    getOwnPropertyDescriptor<T extends object, P extends PropertyKey>(
-        target: T,
-        propertyKey: P,
-    ): (P extends keyof T ? TypedPropertyDescriptor<T[P]> : PropertyDescriptor) | undefined;
-};
+const { getOwnPropertyDescriptor: $gOPD } = Reflect;
 const { getOwnPropertyNames: $gOPN } = Object;
+const hasOwn = Function.prototype.call.bind(Object.prototype.hasOwnProperty) as typeof import('has');
```

Comment:
if the type for this is available why not just use the package?

Reply:
Because **DefinitelyTyped** doesn’t have it as a dependency.

Reply:
It could add it tho, it’s just a dev dep